react/containers/LoginPage/view.jsx

```diff
@@ -13,26 +17,30 @@ class PublishPage extends React.Component {
     }
   }
   render () {
+    const pageTitle = createPageTitle('Login');
+    const canonicalLink = createBasicCanonicalLink('login');
+    const metaTags = createBasicMetaTags();
```

Comment:
You can move all of these utility functions into the `SEO` component. Just pass in the minimum values needed and create the actual title/link/tags inside the component.

Reply:
Then you don't need to add the same three lines to every page component.
`<SEO title="Login" link="login"  />` 
or
`<SEO title="Channel" link="channel"  channel={channel} />` 

Then in the component you can check if `channel` or `asset` was passed in and build the meta tags accordingly. 
```
if (!props.asset || !props.channel) {
    metaTags = createBasicMetaTags();
}
```